src/spatial/models/sentinel2_context_model.py

```
@@ -228,30 +243,49 @@ def get_context(self, latitude, longitude, start_date=None, end_date=None):
+            for item in items:
+                try:
+                    return finalize_result(
+                        self._sample_item(
+                            item,
+                            latitude,
+                            longitude,
+                            allow_cloudy_point=True,
+                        )
+                    )
+                except Exception:
+                    continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)